main/Modal/ModalPicker.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { Picker } from 'react-native';
+import styled from 'styled-components/native';
+
+interface ModalPickerProps {
+  value?: string | undefined;
+  setValue: React.Dispatch<React.SetStateAction<boolean>>;
+  setIsAnimated: React.Dispatch<React.SetStateAction<boolean>>;
+  data?: [] | undefined;
+}
+
+function ModalPicker(props: ModalPickerProps): React.ReactElement {
+  const [options, setOptions] = useState(props.data);
+
+  const { value, setValue, setIsAnimated } = props;
+
+  return (
+    <>
+      <Picker
+        selectedValue={value}
+        onValueChange={(itemValue, itemIndex) => {
+          setValue(itemValue);
+          alert('Selected!');
+          setIsAnimated(false);
```

Comment:
After selecting any name on ModalPicker I get this error:

<img width="1297" alt="스크린샷 2020-07-15 17 33 48" src="https://user-images.githubusercontent.com/50701501/87524088-b6242a80-c6c2-11ea-9b57-6d6afe512f51.png">

Reply:
`setIsAnimated` is never sent on `props`.